### PR TITLE
feat(alert): Styles udpates - OEL-1391

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: src/themes/default/src/scss/base/_utilities.scss [hunk 1]
```diff
@@ -63,13 +63,10 @@ $utilities: map-merge(
   )
 );
 
+/* stylelint-disable no-descending-specificity */
 a {
   text-underline-offset: 2px;
 
-  &:visited {
```
Comment (tibi2303) [inline src/themes/default/src/scss/base/_utilities.scss:69]: They are not taking effect if they styles for visited hover are set on top, so i needed to put them at the bottom but the lint was giving the error and i needed to disable it.

File: src/themes/default/src/scss/_alert.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.alert {
+  .alert-content {
+    color: $dark;
+  }
+}
+@each $state, $value in $theme-colors {
```
Comment (planctus) [inline src/themes/default/src/scss/_alert.scss:6]: why do we need this..? My assumption was that we would have just been using the bs mixin, only needing to change the value of a few vars..

Comment (tibi2303) [reply src/themes/default/src/scss/_alert.scss:6]: in figma, the text color is based on the variant... no other way

File: src/themes/default/src/scss/_alert.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.alert {
```
Comment (planctus) [inline src/themes/default/src/scss/_alert.scss:1]: In my idea, we would not have needed a custom scss for applying these changes, isn't this something we could do with a utility..?

File: src/themes/default/src/scss/_alert.scss (hunk 2 above)
Comment (tibi2303) [reply src/themes/default/src/scss/_alert.scss:6]: i will do the approach as you mentioned below, i will add the icon color in template directly.

File: src/themes/default/src/scss/_alert.scss (hunk 3 above)
Comment (tibi2303) [reply src/themes/default/src/scss/_alert.scss:1]: indeed, thank you, i will change this

File: src/themes/default/src/scss/_alert.scss (hunk 2 above)
Comment (planctus) [reply src/themes/default/src/scss/_alert.scss:6]: i'm not sure, isn't the icon getting the theme color..? couldn't you pass text-{themeColor} class to the icon instead?

**Review by planctus — APPROVED** [review]

**Pull request merged**